pkg/storage/key_storage.go

```diff
@@ -205,3 +206,19 @@ func (k *KeyStorage) RandomAddress(ctx context.Context) (string, error) {
 
 	return addresses[rand.Intn(len(addresses))], nil
 }
+
+func (k *KeyStorage) ImportAccounts(ctx context.Context, accounts []configuration.PrefundedAccount) error {
+	// Import prefunded account and save to database
+	for _, acc := range accounts {
+		keyPair, err := keys.ImportPrivKey(acc.PrivateKeyHex, acc.CurveType)
+		if err != nil {
+			return fmt.Errorf("%w: unable to import prefunded account", err)
+		}
+
+		err = k.Store(ctx, acc.Address, keyPair)
```

Comment:
You may want to write a comment that this will error if the key already exists.

Reply:
@patrick-ogrady I changed this logic to skip importing this key if it already exists instead of erroring.